server/src/test/java/io/crate/role/TransportRoleActionTest.java

```diff
@@ -49,23 +49,29 @@ public class TransportRoleActionTest extends ESTestCase {
     @Test
     public void testCreateFirstUser() throws Exception {
         Metadata.Builder mdBuilder = new Metadata.Builder();
-        TransportCreateRoleAction.putRole(mdBuilder, "root", true, null);
+        TransportCreateRoleAction.putRole(mdBuilder,
+            "root",
+            true,
+            null,
+            Map.of("iss", "https:dummy.org", "username", "test"));
         RolesMetadata metadata = (RolesMetadata) mdBuilder.getCustom(RolesMetadata.TYPE);
         assertThat(metadata.roleNames()).containsExactly("root");
+        assertThat(metadata.roles().get("root").jwtProperties())
+            .containsExactlyInAnyOrderEntriesOf(Map.of("iss", "https:dummy.org", "username", "test"));
```

Comment:
We stream and serialise as Map, order is not important. 
Does it worth having a strict order?